browser-test/src/northstar_header.test.ts

```diff
@@ -0,0 +1,150 @@
+import {test, expect} from './support/civiform_fixtures'
+import {
+  loginAsAdmin,
+  loginAsTestUser,
+  logout,
+  validateScreenshot,
+  validateAccessibility,
+  enableFeatureFlag,
+  seedProgramsAndCategories,
+} from './support'
+
+test.describe('Header', {tag: ['@northstar']}, () => {
+  /**
+   * @todo (#4360) add a "Not logged in, guest mode disabled" test once we can get to the programs page without logging in, for an entity without guest mode.
+   */
+  test('Check screenshots and validate accessibility', async ({
```

Comment:
(blocking) Two thing.

1) Not all the tests are enabling the northstar feature flag. You can use `test.beforeEach` to just code it once and Playwright will run the code before each test.

2) Since everything is within the `test.describe` that already has the `@northstar` tag, you can remove the test level ones.

Reply:
Thanks! Done.